cmd/opampsupervisor/.goreleaser.yml

```
@@ -0,0 +1,140 @@
+before:
+  hooks:
+    - go mod download
+monorepo:
+  tag_prefix: cmd/opampsupervisor/
+  dir: .contrib/cmd/opampsupervisor
+version: 2
+builds:
+  - flags:
+      - -trimpath
+    ldflags:
+      - -s -w -X github.com/open-telemetry/opentelemetry-collector-contrib/cmd/opampsupervisor/internal.version={{ .Version }}
+    env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - windows
+      - darwin
+    goarch:
+      - amd64
+      - arm64
+      - ppc64le
+    ignore:
+      - goos: windows
+        goarch: arm64
```

you can also ignore ppc64le for every OS except linux, it's not supported by Go anyways and will not build anything.
([source](https://go.dev/doc/install/source#environment:~:text=ppc64-,linux,ppc64le,-linux))

```suggestion
        goarch: arm64
      - goos: windows
        goarch: ppc64le
      - goos: darwin
        goarch: ppc64le
```